plugins/webp-uploads/picture-element.php

```diff
@@ -141,14 +140,11 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 	$filter = static function (): bool {
 		return false;
 	};
-	add_filter( 'wp_calculate_image_srcset_meta', $filter );
-	$original_image_without_srcset = wp_get_attachment_image( $attachment_id, $original_sizes, false, array( 'src' => $original_image ) );
-	remove_filter( 'wp_calculate_image_srcset_meta', $filter );
 
 	return sprintf(
 		'<picture class="%s" style="display: contents;">%s%s</picture>',
 		esc_attr( 'wp-picture-' . $attachment_id ),
 		$picture_sources,
-		$original_image_without_srcset
+		$image
```

Comment:
@adamsilverstein As a fallback image, we should return the original image. In [this MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_type_attribute), they return the original image instead of mime-type-specific images.

I couldn't find proper documentation for this. What do you think?

Reply:
Yes, I think this is right. 

Also, I think the `source` being generated for `'image/jpeg'` can also be removed as well, because if the browser doesn't support AVIF/WebP then it can just fall back to the `img` which is a JPEG.

Reply:
https://github.com/WordPress/performance/blob/1ed2fbdab5fa47e2b174b813f6d5d3273b589482/plugins/webp-uploads/picture-element.php#L56-L64

Should we remove `line 61` for the same.

Reply:
I think we can open a separate PR for this issue to avoid complicating the current one. This will allow us to address it more effectively.